x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/rest/action/RestAuthenticateActionTests.java

```diff
@@ -101,6 +113,11 @@ public void testAuthenticateApi() throws Exception {
             assertThat(roles.size(), is(1));
             assertThat(roles, contains(SecuritySettingsSource.TEST_ROLE));
         }
+        if (operatorUser) {
+            assertThat(objectPath.evaluate("operator"), equalTo(true));
+        } else {
+            assertThat(objectPath.evaluate("operator"), equalTo(null));
+        }
```

Comment:
I think we also need a test case where operator privileges is enabled but the user is not an operator and assert the new field does not present.